Default.sublime-commands

```diff
@@ -24,6 +24,12 @@
         "command": "sublime_linter_toggle_highlights"
 
     },
+    {
+        "caption": "SublimeLinter: Fix By Ignoring",
```

Comment:
I'm still partial to the phrase "SublimeLinter: Ignore Error at Cursor", because ignoring `!=` fixing, and I think it's clearer about what's going to happen.

I'd also add this to the context menu, because it makes sense to have it there "in context":

```json
[
    {
        "id": "sublimelinter",
        "caption": "Linter",
        "children": [
            {
                "id": "sublimelinter-lint",
                "caption": "Lint This View",
                "command": "sublime_linter_lint"
            },
            {
                "id": "sublimelinter-fix",
                "caption": "Ignore Error at Cursor",
                "command": "sl_fix_by_ignoring"
            }
        ]
    }
]
```

Reply:
For the context menu, we had an issue that we should *not* add anyting to it.  And although we didn't follow the suggestion "for reasons", I'm actually behind it.  We should not mess with the context menu IMO. (And for example Sublime Text itself should *not*, **really not** add all these Sublime Merge ads here.  It's not adware or shareware.  What huge stupid decision is that?  What users think of other software doing this?) Instead we should add this to the README.

"Fix by ignoring" was always only playful for the draft version here.  But I'd go with the neutral "SublimeLinter: Quick Actions".

Reply:
> We should not mess with the context menu IMO

Fair enough. The thing is nog enough as it is. However, we already have an entry in there, and this expands that single entry to expose multiple features. So, it's not making the situation worse IMO.

I don't think the context menu is an important part of interacting with ST, but I think this entry belongs there. Or we backtrack and remove the `Lint This View` entry. I could get behind that for a big release like we're planning here (this is at least a 4.5 if not a 5.0).

I totally agree about the Sublime Merge thing by the way. At least have the decency to group them so we don't have 4 entries sitting there doing nothing. 🤷‍♂️